softdevice_controller/CHANGELOG.rst

```diff
@@ -62,8 +62,9 @@ Bug fixes
     * The central-like scheduling activity required less than 1 ms to complete at the point in time where the scanner started.
 * Fixed a rare issue where the scanner would be stuck in the synchronizing state after failing to receive an ``AUX_ADV_IND`` packet.
   This could only happen when the corresponding ``ADV_EXT_IND`` packet contained a resolvable address, private address resolution is enabled, and the periodic advertising list is not used. (DRGN-22230)
-* Fixed an issue where the controller could generate the LE Advertising Set Terminated event one event sooner than expected. (DRGN-22705)
-  This could only happen when (all of the following)
+* Fixed an issue where the controller could generate the LE Advertising Set Terminated event one event sooner than expected (DRGN-22705).
+  This could only happen when (all of the following):
```

Comment:
This change looks unintentional? It will anyway be overwritten next time the softdevice controller is exported.

Reply:
Ok, we will need to backport it to the internal repo then